server/routes/helpers/database_singleton.js

```diff
@@ -6,12 +6,11 @@ var DB_URI = process.env.DATABASE_URI || require('../../config/secret').DATABASE
 var db;
 
 function getDbInstance() {
-  if (!db) {
-    console.log("About to connect to DB_URI: "  + DB_URI );
-    mongoose.connect(DB_URI);
-    db = mongoose.connection;
-  }
```

Comment:
This block will be invoked even database isn't connected because db was init in https://github.com/utsab/fcc-classroom-mode/pull/17/files#diff-3d733a7237d8f1c7811b400827720e80L6.